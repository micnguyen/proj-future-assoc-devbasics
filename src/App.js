import logo from './logo.svg';
import './App.css';
import { MichaelNguyen } from './components/MichaelNguyen/MichaelNguyen';
import { AdamSchlicht } from './components/AdamSchlicht/AdamSchlicht';
import { EthanMoliver } from "./components/EthanMoliver/EthanMoliver";
import { DanielChegwidden } from './components/DanielChegwidden/DanielChegwidden';
import { RuhaabSheikh } from "./components/RuhaabSheikh/RuhaabSheikh";
import { ChristianTorrisi } from './components/ChristianTorrisi/ChristianTorrisi';

import { NauvooPerez } from './components/NauvooPerez/NauvooPerez';
import { OllyWilson } from "./components/OllyWilson/OllyWilson";

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>Name | Mantel Brand | Favourite Food | Last show watched</p>
        <MichaelNguyen />
        <AdamSchlicht /> 
        <EthanMoliver />
        <DanielChegwidden /> 
        <RuhaabSheikh />
        <MichaelNguyen /> 
        <ChristianTorrisi /> 
        <NauvooPerez /> 
        <OllyWilson />
      </header>
    </div>
  );
}

export default App;
