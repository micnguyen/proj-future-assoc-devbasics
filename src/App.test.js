import { render, screen } from '@testing-library/react';
import App from './App';

test('heading should always be there', () => {
  render(<App />);
  const heading = screen.getByText(/Name | Mantel Brand | Favourite Food | Last show watched/i);
  expect(heading).toBeInTheDocument();
});

test('page renders correctly', () => {
  render(<App />);
  const page = render.create(<App />).toJSON();
  expect(page).toMatchSnapshot();
});