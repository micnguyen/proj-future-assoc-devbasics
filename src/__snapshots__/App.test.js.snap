// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Name | Mantel Brand | Favourite Food | Last show watched
    </p>
    <p>
      Michael Nguyen | DigIO | Steak | Tinder Swindler
    </p>
    <p>
      Colin Loh | Azenix | Sushi | Rouge One
    </p>
    <p>
      Michael Nguyen | DigIO | Steak | Tinder Swindler
    </p>
    <p>
      Adam Schlicht | Kasna | Sultana Bran | Flight of the Conchords
    </p>
    <p>
      Ethan Moliver | CMD | Spicy char kway teow | Community
    </p>
    <p>
      Daniel Chegwidden | CMD | Ramen | Business Proposal
    </p>
    <p>
      Ruhaab Sheikh | CMD | Biryani | Formula 1: Drive to Survive
    </p>
    <p>
      Nauvoo Perez | Kasna | Currently: Bubble Tea | Miss Fisher's Murder Mysteries
    </p>
  </header>
</div>
`;
